docs/source/reference/kubernetes/examples/kueue-example.rst

```diff
@@ -0,0 +1,315 @@
+.. _kubernetes-example-kueue:
+
+Using SkyPilot with Kueue
+=========================
+
+Kueue is an open source Kubernetes job scheduler.
+This example shows how SkyPilot can be usedwith Kueue to schedule jobs on Kubernetes.
+
+Install Kueue
+-------------
+
+This example assumes a Kubernetes cluster is already set up.
+
+To install Kueue, run the following command:
+
+.. code-block:: bash
+
+    # See https://github.com/kubernetes-sigs/kueue/releases for available versions.
+    VERSION=v0.11.4
+    kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+
+Kueue does not support scheduling plain pods out of the box. Since SkyPilot creates and manages workloads as pods,
+Kueue config needs to be patched to support plain pods.
+
+.. code-block:: bash
+
+    # Extract and patch the config nd save it to /tmp/kueueconfig.yaml
+    # This is required because SkyPilot creates and manages workloads as pods
+    kubectl -n kueue-system get cm kueue-manager-config -o jsonpath={.data.controller_manager_config\\.yaml} | yq '.integrations.frameworks += ["pod"]' > /tmp/kueueconfig.yaml
+    # Apply the changes from /tmp/kueueconfig.yaml created above
+    kubectl -n kueue-system create cm kueue-manager-config --from_file=controller_manager_config.yaml=/tmp/kueueconfig.yaml --dry-run=client -o yaml | kubectl -n kueue-system apply -f -
+    # Restart the kueue-controller-manager pod with the following command
+    kubectl -n kueue-system rollout restart deployment kueue-controller-manager
+    # Wait for the restart to complete
+    kubectl -n kueue-system rollout status deployment kueue-controller-manager
+
+Create multiple namespaces
+--------------------------
+
+Multiple teams may share the same Kubernetes cluster under their own namespaces.
+To simulate this, two namespaces are created: ``team1`` and ``team2``.
+
+.. code-block:: bash
+
+    kubectl create namespace team1
+    kubectl create namespace team2
+
+Create a kueue resource flavor
```

Comment:
```suggestion
Create a Kueue resource flavor
```
May need to patch https://github.com/skypilot-org/skypilot/blob/007c2e532e298f412319e6241d5f243e947689a5/docs/source/extension/linting.py#L6-L7
Applies to other headers as well